justfile

```diff
@@ -5,7 +5,7 @@ demo *args:
     docker compose up {{args}}
 
 demo-native:
-    cargo build --release
+    cargo build --release --features testing
```

Comment:
Not clear on why we need this now

Reply:
Yeah forgot to remove this. Hid this behind testing feature first but then realised it won't work for the docker images.